ksql-examples/src/main/java/io/confluent/ksql/datagen/DataGen.java

```diff
@@ -171,19 +217,22 @@ private static void usage() {
               .put("format", (builder, argVal) -> builder.valueFormat = parseFormat(argVal))
               .put("topic", (builder, argVal) -> builder.topicName = argVal)
               .put("key", (builder, argVal) -> builder.keyName = argVal)
-              .put("iterations", (builder, argVal) -> builder.iterations = parseIterations(argVal))
+              .put("iterations", (builder, argVal) -> builder.iterations = parseInt(argVal, 1))
               .put("maxInterval",
-                  (builder, argVal) -> builder.maxInterval = parseIterations(argVal))
+                  (builder, argVal) -> builder.maxInterval = parseInt(argVal, 0))
               .put("schemaRegistryUrl", (builder, argVal) -> builder.schemaRegistryUrl = argVal)
               .put("propertiesFile",
-                  (builder, argVal) -> builder.propertiesFile = toFileInputStream(argVal))
+                  (builder, argVal) -> builder.propertiesFile = toFileInputStream(argVal).get())
+              .put("msgRate", (builder, argVal) -> builder.msgRate = parseMsgRate(argVal))
```

Comment:
Looks like you intended to replace this (and `parseNumThreads` below) with `parseInt`?